src/guide/using-the-cli.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Using the CLI
+type: guide
+order: 5
+---
+
+Creating and running labs on [machinelabs.ai](https://machinelabs.ai) is great. Sometimes however, we prefer to start our experiments on our local machines, or, we have an existing project that we'd like to import into MachineLabs to run it there.
+
+The MachineLabs CLI makes it easy to create new labs, importing existing projects, as well as pulling labs from the MachineLabs platform - all that without leaving your terminal!
```

Comment:
You are the English expert but wouldn't it be "_import_ existing labs" in this case?

Reply:
Yep. big bug.